sniffles/petabi_rule_writer/petabi_rule_writer.py

```diff
@@ -173,14 +149,38 @@ def check_pcre_compile(re):
     return True
 
 
+def formatRule(regexList=None, ruleName=None, proto='tcp', src='any',
+               dst='any', dport='any', sport='any', out_of_order=False,
+               out_of_order_prob=False, packet_loss=False, tcpOverlap=False,
+               count='1', fragment=False, flow='to server', split=False,
+               ttl=None, ttlExpiry=False, pktAck=False, trafficAck=False):
+
+    rule = []
+    ruleNo = 0
+    for regex in regexList:
+        if ruleName is None or ruleNo > 0:
```

Comment:
If we are using one name for all the rules, why not just append the number to the name.  If the name is none, then just set it to "Rule #".  That would simplify the above logic.